cmd/find.go

```diff
@@ -257,6 +257,24 @@ func subArgsHelper(args, filepath string, fileContent contentMessage) string {
 		}
 	}
 
+	for {
```

Comment:
These two for loops should be combined.. and created in a concise manner.. Basically you need to have a parser code which can execute a function as well instead.  

Also not sure why would you need an infinite for loop.